ruby_connectors/spec/salesforce_connector/reader_spec.rb

```
@@ -0,0 +1,32 @@
+require 'vcr'
+
+RSpec.describe RubyConnectors::SalesforceConnector::Reader do
+  let(:valid_config) do
+    {
+      client_id: ENV['SALESFORCE_CLIENT_ID'],
+      client_secret: ENV['SALESFORCE_CLIENT_SECRET'],
+      refresh_token: ENV['SALESFORCE_REFRESH_TOKEN']
+    }
+  end
+
+  context '#read' do
```

RuboCop (1.73)

[convention] Use `describe` for testing methods.

(RSpec/ContextMethod)